.github/workflows/trigger-documentation.yml

```
@@ -0,0 +1,16 @@
+name: Trigger doc rebuild
+
+on:
+  push:
+    branches:
+      - master
+  paths:
```

actionlint (1.7.12)

[error] unknown Webhook event "paths". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)